Framework/Geometry/test/XMLInstrumentParameterTest.h

```diff
@@ -140,13 +153,14 @@ class XMLInstrumentParameterTest : public CxxTest::TestSuite {
     series.addValue("2000-11-30T01:02:00", 5);
 
     XMLInstrumentParameter_sptr logFile = make_logfile_object("median");
-    const double actualFilteredValue = logFile->createParamValue(&series);
+    const double actualFilteredValue = logFile->createParamValue(&series, roi);
     TSM_ASSERT_EQUALS("Filtering by Median is not performed correctly", expectedFilteredValue, actualFilteredValue);
   }
 
   // This functionality will soon be legacy, since filtering by nth-position is
   // not a good idea.
   void test_filter_by_nth_position() {
+    Kernel::TimeROI *roi = nullptr;
```

Comment:
I think that if the ROI excludes the first value, `position 2` should correspond to the value `2`. Can you please test with ROI `[02, 04]`?